using_images/s2i_images/dot_net_core.adoc

```diff
@@ -41,18 +42,18 @@ ifdef::openshift-enterprise[]
 and {product-title} versions 3.3 and later
 endif::openshift-enterprise[]
 
-The .NET Core software collection for version 2.0 supports the
+The .NET Core software collection for versions 2.1 and 2.0 supports the
```

Comment:
Can you please put this paragraph below the next one? It has become less important.
You can remove the first sentence, so it is clear the paragraph only applies to Versions 1.0 and 1.1.